src/components/Configurator.ts

```diff
@@ -1,42 +1,84 @@
-import path from 'path';
 import fs from 'fs';
 import yaml from 'js-yaml';
 
 export default class Configurator {
 
   path = '';
-  config = {} as any;
+  config: any = {};
```

Comment:
I dont like the any types, but I really have NO CLUE how to do nested types:
{
  [key: string] : {
     [key: string] : {... } 
  }
}

especially if you dont know how deep you can go